auditorium/views/decorators/with-layout.test.js

```diff
@@ -22,7 +22,7 @@ describe('src/decorators/with-layout.js', function () {
     it('wraps the given content in the application layout', function () {
       var wrappedView = withLayout()(testView)
       var result = wrappedView(app.state, app.emit)
-      assert(result.matches('.container'))
+      assert(result.matches('.mw7.center.pa3.sans-serif'))
```

Comment:
As discussed, it's probably better to introduce more `data-role` attributes for this kind of tests.

Reply:
:+1: